### PR TITLE
Expose prediction, covariance prediction methods

### PR DESCRIPTION
**Pull request by joshbeckman**
This is helpful if you would like to show a predicted next value from the filter.

### CODE REVIEW

**Comment by wouterbulten** [comment]
Hi @andjosh, thanks for your pull request. Could you give some insights in the use case of the change? The 'predict' function is merely the result of the control and the movement model.

**Comment by joshbeckman** [comment]
I was using this library to track latency in some JS animations. I thought it would be interesting to predict the next latency value before performing the next animation sequence and use that to schedule such that the animation didn't increase latency in the JS context.

File: src/kalman.js [hunk 1]
```diff
@@ -60,6 +60,23 @@ export default class KalmanFilter {
...
+  */
+  predict(u = 0) {
+    return (this.A * this.x) + (this.B * u);
+  }
+  
+  /**
+  * Predict next covariance
+  * @return {Number}
+  */
+  predictCovariance() {
```
Comment (wouterbulten) [inline src/kalman.js]: Could you change the docblock and the name of this function? The covariance here describes the uncertainty of the prediction, so we are not really 'predicting' the covariance. To keep it in style of the other (simply named) functions, 'uncertainty()' would probably be clear enough.

**Review by wouterbulten — CHANGES_REQUESTED** [review]
Hi! My apologies for the late reply. I've reviewed the code and thought about the use case. I think it makes sense and I would be open to add it to the repo. I have only one comment that I would like to be addressed.

Second, I want to license the new version under the MIT license. Are you fine with this?

File: src/kalman.js (hunk 1 above)
Comment (joshbeckman) [reply src/kalman.js]: That sounds like a great idea!

**Comment by joshbeckman** [comment]
Changed.

**Review by wouterbulten — APPROVED** [review]

**Comment by wouterbulten** [comment]
Thanks, I will merge the branch later this week.

**Comment by wouterbulten** [comment]
Merged, will be part of next release.

**Pull request merged**